plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/AbstractKuduConnectorTest.java

```diff
@@ -292,6 +316,22 @@ public void testCreateTableWithColumnComment()
         assertUpdate("DROP TABLE IF EXISTS " + tableName);
     }
 
+    @Override
+    public void testCreateTableAsSelectWithTableComment()
+    {
+        String tableName = "test_create_" + randomTableSuffix();
+
+        if (getKuduServerVersion().equals(EARLIEST_TAG)) {
+            assertUpdate("CREATE TABLE " + tableName + " COMMENT 'test comment' AS SELECT name FROM nation", 25);
+            assertNull(getTableComment(getSession().getCatalog().orElseThrow(), getSession().getSchema().orElseThrow(), tableName));
```

Comment:
Could you look for the way to identify if the server version allows a comment? CREATE TABLE with comment in EARLIEST_TAG should throws an exception instead of ignoring silently.

Reply:
I'm wondering why `tableMetadata.getComment().ifPresent(options::setComment)` doesn't throw an exception if the kudu version does not support this?

I can't seem to find any method within kudu to retrieve the server version. I'll keep looking but maybe checking if the method actually executed with a comment present by checking the CreateTableOptions and throwing an exception if nothing happened after the `setComment` can help.